hie.yaml

```diff
@@ -90,7 +90,7 @@ cradle:
     - path: "instrumentation/yesod/test"
       component: "hs-opentelemetry-instrumentation-yesod:test:hs-opentelemetry-instrumentation-yesod-test"
 
-    - path: "otlp/src"
+    - path: "otlp/./"
```

Comment:
I changed it to `otlp/`, re-ran `gen-hie > hie.yaml`, and it replaced it with `otlp/./` again. I will defer to `gen-hie` because every time we re-run `gen-hie` it will just update this again.